scripts/check_deployed_market.py

```diff
@@ -0,0 +1,162 @@
+from brownie import accounts, Contract
+import numpy as np
+from pytest import approx
+
+OVL_ADDR = '0x4305C4Bc521B052F17d389c2Fe9d37caBeB70d54'
+STATE_ADDR = '0xC3cB99652111e7828f38544E3e94c714D8F9a51a'
+MARKET_ADDR = '0x1067B7DF86552A53D816CE3fed50d6D01310b48f'
+FEED_ADDR = '0x6C4296156875dAb08Dd35A8E539E0aef6B88f459'
+PARAMS = {
+    'k': 680187455489,
+    'lambda': 1101276756574031744,
+    'delta': 3825444954737599,
+    'capPayoff': 10000000000000000000,
+    'capNotional': 800000000000000000000000,
+    'capLeverage': 5000000000000000000,
+    'circuitBreakerWindow': 2592000,
+    'circuitBreakerMintTarget': 66670000000000000000000,
+    'maintenanceMarginFraction': 77621507357346512,
+    'maintenanceMarginBurnRate': 64852413396413568,
+    'liquidationFeeRate': 50000000000000000,
+    'tradingFeeRate': 750000000000000,
+    'minCollateral': 100000000000000,
+    'priceDriftUpperLimit': 100000000000000
+}
+
+
+def load_contract(address):
+    try:
+        return Contract(address)
+    except ValueError:
```

Comment:
Is the only ValueError possible here the error you are catching? Could a different ValueError be raised, leading to incorrect error handling?

Reply:
Valid point. But I'm not really trying to catch an error here per se. And for what it's worth, no, not expecting any other kind of ValueError here either.
`Contract.from_explorer(address)` reads a contract from etherscan (takes 3-4 secs). But if it has already been read once, brownie caches it and subsequently it can be loaded just with `Contract(address)` (much faster). 
I've copied this code snippet from curve finance's brownie scripts.